backend/src/app/rpc/commands/files_update.clj

```diff
@@ -66,6 +70,15 @@
     [:revn {:min 0} :int]
     [:session-id ::sm/uuid]]])
 
+(sm/def! ::repair-file-params
```

Comment:
Use simple def instead of global registry

Reply:
deleted